<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QTimeDisplayGeneralDLg</class>
 <widget class="QWidget" name="QTimeDisplayGeneralDLg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>399</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="m_formatcmb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>176</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>HH:MM</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HH:MM:SS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HH-MM-SS</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>HH-MM</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>176</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>格    式：</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>291</y>
     <width>361</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>排列</string>
   </property>
   <widget class="QRadioButton" name="m_leftradio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>居左</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_middleradio">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>居中</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="m_rightradio">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>居右</string>
    </property>
   </widget>
  </widget>
  <widget class="QComboBox" name="m_fontcmb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>203</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>11</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_patterncmb">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>140</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>142</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>样  式：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_20">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>231</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字体大小：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>203</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>字    体：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="ID">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="LoadShape">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>10</y>
     <width>211</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>外形</string>
   </property>
  </widget>
  <widget class="QComboBox" name="m_fontsizecmb">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>231</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QGraphicsView" name="LView">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>131</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Frame_Color">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>42</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>43</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>边框色：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>168</x>
     <y>76</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>前景色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="FG_Color">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>75</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BG_Color">
   <property name="geometry">
    <rect>
     <x>229</x>
     <y>107</y>
     <width>151</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>167</x>
     <y>109</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>背景色：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Text_Color">
   <property name="geometry">
    <rect>
     <x>89</x>
     <y>259</y>
     <width>191</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>262</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>文 本 色：</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>ID</tabstop>
  <tabstop>LoadShape</tabstop>
  <tabstop>LView</tabstop>
  <tabstop>Frame_Color</tabstop>
  <tabstop>FG_Color</tabstop>
  <tabstop>BG_Color</tabstop>
  <tabstop>m_patterncmb</tabstop>
  <tabstop>m_formatcmb</tabstop>
  <tabstop>m_fontcmb</tabstop>
  <tabstop>m_fontsizecmb</tabstop>
  <tabstop>Text_Color</tabstop>
  <tabstop>m_leftradio</tabstop>
  <tabstop>m_middleradio</tabstop>
  <tabstop>m_rightradio</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
